jaeger_client/codecs.py

```diff
@@ -232,3 +235,50 @@ def extract(self, carrier):
         return SpanContext(trace_id=trace_id, span_id=span_id,
                            parent_id=parent_id, flags=flags,
                            baggage=None)
+
+
+def header_to_hex(header):
+    if not isinstance(header, (str, unicode)):
+        return None
+    return int(header, 16)
```

Comment:
note that invalid strings will cause ValueError, which breaks the contract of the codec. The main codec protects against that.